knative-serving/scalability-and-performance-serving.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="scalability-and-performance-serving"]
+= Scalability and performance of OpenShift Serverless Serving
+:context: scalability-and-performance-serving
+
+toc::[]
+
+OpenShift Serverless consists of several different components which have different resource requirements and scaling behaviors. These components are horizontally and vertically scalable, but their resource requirements and configuration highly depend on the actual use-case. 
```

Comment:
```suggestion
OpenShift Serverless consists of several different components that have different resource requirements and scaling behaviors. These components are horizontally and vertically scalable, but their resource requirements and configuration highly depend on the actual use-case. 
```
See ISG quick reference: https://spaces.redhat.com/display/CCSENABLE/Commas+in+restrictive+and+nonrestrictive+clauses